### PR TITLE
Update serialization format version

### PR DESCRIPTION
**Pull request by dslmeinte**

### CODE REVIEW

File: src-pkg/serialization.ts [hunk 1]
```diff
@@ -48,6 +48,12 @@ export type SerializedLanguageReference = {
     version: string
 }
 
+
+/**
+ * The <em>current</em> version of the serialization format.
+ */
+export const serializationFormatVersion = "2023.1"
```
Comment (ftomassetti) [inline src-pkg/serialization.ts:55]: Maybe I would call it "currentSerializationFormatVersion"

**Review by ftomassetti — APPROVED** [review]
Looks good

File: models/instances/library.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "serializationFormatVersion": "1",
```
Comment (enikao) [inline models/instances/library.json:2]: I wouldn't do this yet -- there might be more changes.
I already know of one (meta)-change: This field (`serializationFormatVersion`) is currently spec'd as "integer" but will change to "string"

Comment (dslmeinte) [reply models/instances/library.json:2]: In the TS and Java implementations this is already a string. I guess we're slightly ahead of the updating of the specification?

Also: these files are not leading in any way.

**Pull request merged**